src/ProofOfConcepts/Chips/ConsumerGroupLabel.js

```diff
@@ -0,0 +1,26 @@
+import "@patternfly/react-core/dist/styles/base.css";
```

Comment:
You should never import PatternFly's css directly, those are already available for you in Storybook.

Reply:
Thanks for letting me know! I removed all unnecessary CSS imports in the new PR.